### DISCUSSION TITLE
Deserializing the manifest

### DISCUSSION THREAD

**Discussion by raffaeler** [microsoft/aspire#6328] [Q&A] [answered]
Hi I could find the serialization code for the manifest, but not the deserializing one.
I need to deserialize the manifest in another application, possibly reusing the aspire model. I need it to extract key information before publishing.
Can someone point me to the deserialization please?

**Comment by davidfowl** [comment, marked as answer]
There's no code in this project for processing the manifest. Today that is done by deployment tools. Aspirate ships one here https://github.com/prom3theu5/aspirational-manifests/tree/main/src/Aspirate.Processors

**Comment by raffaeler** [comment]
Thank you David, I know Aspirate and it is indeed a great tool.
Anyway, while I understand that Aspire is specific to the inner loop, I believe it should provide everything possible to prepare the deployments using external tools.
I will open a separate discussion for this.
